GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -150,6 +150,11 @@ public sealed partial class RevisionGridControl : GitModuleControl, IScriptHostC
         /// </summary>
         internal Dictionary<ObjectId, string>? FilePathByObjectId { get; set; } = null;
 
+        /// <summary>
+        /// Executed on hotkey Ctrl+A, which is named SelectFirstDiffGroupChanges.
+        /// </summary>
+        internal Action SelectAll { get; set; }
```

Comment:
Any objections?

Reply:
(Sorry, my brain shortcircuited when trying to review this last time, will try the next days...)
I would prefer to hardcode Ctrl-A in RevGrid rather than adding a hotkey for another form, but has not realized if that can be done.

Reply:
> I would prefer to hardcode `Ctrl+A` in RevGrid rather than adding a hotkey for another form, but have not realized if that can be done.

I found different a little less spaghetti variant. FormBrowse still needs to know about `Ctrl+A`.
But the new `GitExtensionsFormBase.ProcessHotkey` is nearly the same as the `GitExtensionsControl.ProcessHotkey` factored out in #9827 - good.